drivers/spi/spi_max32.c

```diff
@@ -242,19 +272,45 @@ static int spi_max32_transceive(const struct device *dev)
 	dfs_shift = spi_max32_get_dfs_shift(ctx);
 
 	len = spi_context_max_continuous_chunk(ctx);
+#ifdef CONFIG_SPI_RTIO
+	if (sqe->op == RTIO_OP_TINY_TX) {
```

Comment:
missing a switch/case here perhaps to cover RX/TX/TINY_TX/TXRX?

Reply:
Hi Tom, I agree on this part, its now split to have switch when using RTIO config and old code. I think now its more readable, if you can take a look and let me know if this is ok?